pkg/agents/randomAgent/agent.go

```diff
@@ -0,0 +1,25 @@
+package agentRandom
```

Comment:
LGTM! Otherwise can we rename just to have consistency?
```suggestion
package randomAgent
```